src/constants/apiPaths.ts

```diff
@@ -3,8 +3,8 @@ const API_PATHS = {
   product: 'https://zfhk5qobji.execute-api.eu-west-1.amazonaws.com/dev',
   order: 'https://.execute-api.eu-west-1.amazonaws.com/dev',
   import: 'https://hrdhhb2s9a.execute-api.eu-west-1.amazonaws.com/dev',
-  // bff: 'https://.execute-api.eu-west-1.amazonaws.com/dev'
   bff: 'https://zfhk5qobji.execute-api.eu-west-1.amazonaws.com/dev',
+  cart: 'http://lexx129-cart-api-develop.us-east-1.elasticbeanstalk.com/api/profile/cart',
```

Comment:
profile/cart is doubled here and in the requests:
await axios.put(`${API_PATHS.cart}/profile/cart`, ...

Reply:
@lexx129 update please this issue

Reply:
Fixed this, thanx guys